packages/nodes-base/nodes/Hubspot/Hubspot.node.ts

```diff
@@ -935,8 +1026,206 @@ export class Hubspot implements INodeType {
 		const resource = this.getNodeParameter('resource', 0) as string;
 		const operation = this.getNodeParameter('operation', 0) as string;
 
-		//https://legacydocs.hubspot.com/docs/methods/lists/contact-lists-overview
-		if (resource === 'contactList') {
+		if (resource === 'customObject' && operation.includes('batch')) {
+			const customObjectType = this.getNodeParameter('customObjectType', 0, '') as string;
+			if (!this.continueOnFail()) {
+				throw new Error('Batch operations only work with continue on fail set to true.');
+			}
```

Comment:
When an element in a batch operation fails, the other elements still get processed. It seems to be common practice in most n8n nodes, to stop execution on error and not process any item after the erroring one.
The batch operations do process the upcoming items (because requests are batched), so the behaviour is not quite what the user would expect, when `continueOnFail` is disabled.